saleor/graphql/account/tests/mutations/staff/test_address_create.py

```diff
@@ -50,7 +60,12 @@ def test_create_address_mutation(
     data = content["data"]["addressCreate"]
     assert data["address"]["city"] == "DUMMY"
     assert data["address"]["country"]["code"] == "PL"
+    assert data["address"]["metadata"] == [{"key": "key", "value": "public"}]
+    assert data["address"]["privateMetadata"] == [{"key": "key", "value": "private"}]
```

Comment:
Instead of copying it in both places we could assign it to the variable and reuse

Reply:
Similar in other tests